graylog2-web-interface/src/components/search/MessageField.jsx

```diff
@@ -40,14 +32,15 @@ const MessageField = React.createClass({
 
     return (
       <span>
-        <dt key={`${key}Title`}>{key} {this._decorationMarker(key)}</dt>
```

Comment:
The creative director (aka @mariussturm) and I were looking into some alternatives and the current approach is the one we liked the most: The idea was to be non-obtrusive when normally looking around message fields, as mostly you don't care if fields are decorated or not, while still having the information somewhere if you really needed to know. Having the `(decorated)` marker right next to the field may be too distracting, specially when several fields are modified by decorators.